<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorEngineRangeDefinitionDialog</class>
 <widget class="QDialog" name="ColorEngineRangeDefinitionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>465</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>418</width>
    <height>424</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Color Range Definition</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../App/rsc/espina.qrc">
    <normaloff>:/espina/espina.svg</normaloff>:/espina/espina.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="m_gridLayout" columnstretch="1,1">
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>From color:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_fromLabel">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="m_fromSpinBox"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>To color:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_toLabel">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="m_toSpinBox"/>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="m_customBar">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Color range:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_showRange">
     <property name="text">
      <string>Show range bar in views.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="m_positionLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Widget position</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_positionCombo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string> Top right corner</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Top left corner</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bottom right corner</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bottom left corner</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="m_orientationLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Orientation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_orientationCombo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Horizontal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vertical</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="m_textPositionLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Title position</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_textPositionCombo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Precede scalar bar</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Succeed scalar bar</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="m_numlabelsLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Number of labels</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="m_labelsSpin">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string> labels</string>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
     <item>
      <widget class="QLabel" name="m_widthLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Relative width</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="m_widthSlider">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1">
     <item>
      <widget class="QLabel" name="m_heightLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Relative height</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="m_heightSlider">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,1">
     <item>
      <widget class="QLabel" name="m_barRatioLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Bar ratio</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="m_barRatioSlider">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="m_decimalsLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Number of decimals</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="m_decimalsSpin">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string> decimals</string>
       </property>
       <property name="value">
        <number>6</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../App/rsc/espina.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ColorEngineRangeDefinitionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ColorEngineRangeDefinitionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
